Bixal/move.mil
# replace Transcom logo with DOD logo in both header & footer

Simple swap of logo used in site header & footer. Per Pivotal ticket [#164551836](https://www.pivotaltracker.com/story/show/164551836).

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Downloaded official DOD svg asset from Wikipedia (per Jared) and placed in the root folder of the site's custom theme
- Swapped out hardcoded references in relevant template files to point to the new file

## Testing

To verify the changes proposed in this pull request…

1. Pull code; clear cache
1. Load the local site's home page and look at the header and footer
1. Repeat step 2 for any internal page

## Screenshots

### Header

Before:
<img width="588" alt="site logo swap - header - before" src="https://user-images.githubusercontent.com/29130580/54232591-58396e80-44e1-11e9-9a54-3342ce697a99.png">

After:
<img width="652" alt="site logo swap - header - after" src="https://user-images.githubusercontent.com/29130580/54232598-5f607c80-44e1-11e9-90ea-a0688c39552c.png">


### Footer

Before:
<img width="260" alt="site logo swap - footer - before" src="https://user-images.githubusercontent.com/29130580/54232614-6ab3a800-44e1-11e9-9d7c-94eaedd63104.png">

After:
<img width="258" alt="site logo swap - footer - after" src="https://user-images.githubusercontent.com/29130580/54232625-70a98900-44e1-11e9-8787-fc314d3a5c81.png">
